src/main/webapp/script.js

```diff
@@ -10,4 +10,55 @@
 // distributed under the License is distributed on an "AS IS" BASIS,
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
-// limitations under the License.
\ No newline at end of file
+// limitations under the License.
+
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(getChart);
+
+const CHART_WIDTH = 800;
+const CHART_HEIGHT = 400;
+
+/**
+ * Fetches data and adds to html
+ */
+function getChart() {
+      const data = new google.visualization.DataTable();
+      data.addColumn('string', 'Range');
+      data.addColumn('number', 'Count');
+      /**
+       * TODO: change hardcoded data to sentiment scores (once that part is finished)
+       */
+        data.addRows([
+          ['-1.0', null],
+          [null, 1],
+          ['-0.8', null],
+          [null, 1],
+          ['-0.6', null],
+          [null, 4],
+          ['-0.4', null],
+          [null, 5],
+          ['-0.2', null],
+          [null, 10],
+          ['0.0', null],
+          [null, 10],
+          ['0.2', null],
+          [null, 10],
+          ['0.4', null],
+          [null, 10],
+          ['0.6', null],
+          [null, 15],
+          ['0.8', null],
+          [null, 15],
+          ['1.0', null]
+        ]);
+
```

Comment:
nit: extra line here?

Reply:
I think it might be too crowded without it? Just because it's seperating 2 different declarations

Reply:
Please refer to this: https://google.github.io/styleguide/jsguide.html#formatting-vertical-whitespace

Reply:
Extra line here could be ok as the code before are all about manipulation on data.
But when in doubt, always follow the style guide :)